pocld/shared_cl_context.cc

```diff
@@ -495,10 +512,10 @@ SharedCLContext::SharedCLContext(cl::Platform *p, unsigned pid,
                                  ReplyQueueThread *s, ReplyQueueThread *f) {
   p->getDevices(CL_DEVICE_TYPE_ALL, &CLDevices);
 
-  cl_context_properties properties[] = {
+  cl_context_properties Properties[] = {
```

Comment:
There's a bunch of other local C++ variables that are getting capitalized in this PR too, what's the logic behind that?

Reply:
I'm unsure about this. PoCL-D was not following the PoCL C++ coding standard (LLVM-style) when it was added and I'm leaning towards converting it step-by-step to the style (like with the rest of the PoCL) code space instead of adding a 3rd coding style just for PoCL-D. But we can do the latter as well if you think it's not worth having mixed styles over "the migration".

Reply:
Nah I was just not aware that LLVM style requires capitalized variable names. Apparently clang-format does not enforce this either. But since it is in the style guide let's keep it in mind and convert the code to adhere to it as we go.